tsfc/fem.py

```diff
@@ -233,6 +234,11 @@ def flat_index(ordered_deriv):
         return result[0]
 
 
+def zero_tracking_literal(table, ufl_element):
+    track_zeros = isinstance(ufl_element, (ufl.EnrichedElement, ufl.MixedElement))
```

Comment:
@wence- I think it's more like "mixed [and vector] elements are constructed in a way that *guarantees* a block of contiguous zero columns".  Then Enriched is there so you catch the HDiv(....) + HDiv(....) elements.  I guess Enriched just passed up information from the subelements rather than doing any inspection of its own.